packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -115,7 +115,7 @@ export default {
 
         showLegacyBanner: {
```

Comment:
I know this wasn't introduced in this PR, but given it's a `boolean`, the naming of the prop should indicate it's a `boolean`. For instance, `shouldShowLegacyBanner`.